util/chunk/row.go

```diff
@@ -220,5 +220,5 @@ func (r Row) GetDatum(colIdx int, tp *types.FieldType) types.Datum {
 
 // IsNull returns if the datum in the chunk.Row is null.
 func (r Row) IsNull(colIdx int) bool {
-	return r.c.columns[colIdx].isNull(r.idx)
+	return r.c.columns[colIdx].IsNull(r.idx)
```

Comment:
we should refactor other `GetXXX` functions binded to `row`.

Reply:
How about doing it in the next PR?
In the next PR, I will introduce `type Sel []uint16` to indicate which rows are selected and refactor some codes of `Row/MutRow`.
And this PR may be big enough.

Reply:
OK